handlers/folders.go

```diff
@@ -47,7 +48,7 @@ func CreateFolder(w http.ResponseWriter, r *http.Request) {
 	// TODO constructor for new objects
 	folder := &models.Folder{
 		SpaceID:   space.ID,
-		Name:      b.Name,
+		Name:      strings.TrimSpace(b.Name),
```

Comment:
Trimming is done automagically is you set `bind.Form(r, &b, bind.Vacuum)` (documentation is still lacking)